### ISSUE TITLE
Description List not handled in docs

### ISSUE THREAD

**Issue by asfalcone** [Kotlin/dokka#2213]
**Describe the bug**
Description List (`<dl>`, `<dt>`, `<dd>`) isn't parsed properly.

In java, the description items are removed. In Kotlin they appear as body text instead of structured DocTag elements

**Expected behaviour**
Dl, Dt, DD appear as structured `DocTag` elements

**Screenshots**
From Kotlin source:
![image](https://user-images.githubusercontent.com/1848683/139303575-26554d11-149c-4198-9e59-a2c89e67d9f3.png)

From Java source:
![image](https://user-images.githubusercontent.com/1848683/139303746-8ea6b809-1b92-4d71-811b-43715abb7027.png)

**To Reproduce**
```
/** 
 * <dl>
 *     <dt>
 *         <code>name="<i>name</i>"</code>
 *     </dt>
 *     <dd>
 *         A URI path segment. To enforce security, this value hides the name of the subdirectory
 *         you're sharing. The subdirectory name for this value is contained in the
 *         <code>path</code> attribute.
 *     </dd>
 *     <dt>
 *         <code>path="<i>path</i>"</code>
 *     </dt>
 *     <dd>
 *         The subdirectory you're sharing. While the <code>name</code> attribute is a URI path
 *         segment, the <code>path</code> value is an actual subdirectory name. Notice that the
 *         value refers to a <b>subdirectory</b>, not an individual file or files. You can't
 *         share a single file by its file name, nor can you specify a subset of files using
 *         wildcards.
 *     </dd>
 * </dl>
 */
 * class Foo
 ```

**Installation**
- Dokka version: 1.6.0-dev-122

**Comment by IgnatBeresnev** [comment]
@asfalcone Hi! I can't tell from `Foo` the class whether it's a kotlin or a java class :) What's the use case? Would you like to be able to make description lists in JavaDocs only or in KDocs as well?

The reason I'm asking is it's unclear to me at the moment whether we should support parsing html tags in KDoc at all.

**Comment by IgnatBeresnev** [comment]
Will be included in the upcoming 1.6.10. The dev version is out.
